### DISCUSSION TITLE
API 3.0 Deprecation - Gazer will not work?

### DISCUSSION THREAD

**Discussion by alec-flatiron** [looker-open-source/looker_deployer#113] [General]
Hi @AdamMinton - we see API v3 will be deprecated in June. When 4.0 became the default, `gzr` broke because of references to spaces or something like that. The fix was [this commit](https://github.com/looker-open-source/gzr/commit/38233991bfc5456ac0cf3d485d12520f50a2ea76) to pin it to v3.1. That code is still in place. Since `gzr` is used by `looker-deployer`, is `looker-deployer` at risk due to the API v3 deprecation? `gzr`'s docs point to using `looker-deployer`, but that seems to be circular.

Thanks,
Alec

**Comment by AdamMinton** [comment]
Hey Alec,

We are planning on upgrading gzr and looker deployer both to API 4.0 in the next couple of weeks. We will be taking the message down from gzr as priorities have shifted on migrating gzr functionality to looker deployer natively.

Thanks!

**Reply by alec-flatiron** [reply]
Excellent, thanks Adam!

**Comment by anyapriya** [comment]
Hi,

Wondering about updates on this since the pr to update looker_deployer to use Looker API 4.0 hasn't been updated in about a month, and we're getting close to June when Looker API 3.X will be removed.  Happy to try to contribute if it would help.  

Thanks!

**Reply by anyapriya** [reply]
Hi @AdamMinton , 

Sorry to tag you, not sure if you got a notification for my post last week, just wondering if you have thoughts on if the updates for looker deployer moving to looker API 4.0 will be ready in time.  Our group is just trying to figure out best next steps in case we need to do a work on our end, and want to leave enough time.  

Thanks so much,
Anya
